lib/theme_check/checks/pagination_size.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+module ThemeCheck
+  class PaginationSize < LiquidCheck
+    severity :suggestion
+    categories :performance
+    doc docs_url(__FILE__)
+
+    attr_reader :max_size
+
+    def initialize(max_size: 50)
+      @max_size = max_size
+    end
```

Comment:
wondering: shouldn't there be a `min_size` as well?  like we would want to warn about anything less than `1` right?